src/main/php/org/bovigo/vfs/vfsStreamWrapper.php

```diff
@@ -292,8 +292,12 @@ public function stream_open(string $path, string $mode, int $options, string $op
 
         $this->mode    = $this->calculateMode($mode, $extended);
         $path          = $this->resolvePath(vfsStream::path($path));
-        $this->content = $this->getContentOfType($path, vfsStreamContent::TYPE_FILE);
-        if (null !== $this->content) {
+        $this->content = null;
```

Comment:
Just out of curiosity again, what's the benefit of assigning this to null and then reassigning it if a value exists?

Reply:
Same as above, to add `/** @var vfsStreamFile|null $content */` so phpstan knows what it is. It doesn't like `/** @var vfsStreamFile|null $this->content */` so assigned it to a local variable first and then the class property.

Without the doc, it sees it as `vfsStreamContent` and a gives an error for method calls like `$this->content->openWithTruncate()` because `openWithTruncate` is not defined on the interface.